pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/fullsecretmanagersecret/create.yaml

```diff
@@ -38,6 +38,7 @@ spec:
   rotation:
     nextRotationTime: "2025-10-02T15:01:23Z"
     rotationPeriod: "3600s"
-  versionAliases:
-    "foo": "111"
-    "bar": "222"
+  # FR to SecretManagerSecretVersion adding aliases.
```

Comment:
Is there a follow-on PR for this (or a bug?)  I'm just wary that because we're removing test coverage, it'll make it harder to track

Reply:
Yup, this is exactly why this PR is "part 1" 😄. I want to handle the secretVersion more carefully and dig deeper into these two resources, because I think something may go wrong...